lib/src/modules/input/InputWithMentions.js

```diff
@@ -94,6 +98,28 @@ export function InputWithMentions({
         />
       </MentionsInput>
       <div ref={mentionsPortalRef} />
+      <ShortcutTrigger
```

Comment:
🔴   why do we need 2 `ShortcutTrigger`s? 

Can't we just set `withCtrlKey` & `withMetaKey` on a single component? 🤔

Reply:
That would require the user to hold both `ctrl` and `meta` and `Enter` to trigger the shortcut.

Reply:
haha oh yea! 😂